File: 2016/gara3/acronimi/att/acronimi.pas
function acronimi(var A, S: string): longint;
var lunghezza: longint;
begin
    (* Per calcolare la lunghezza si puo' fare cosi' *)
    lunghezza := length(A);

    (* Mettete qui il codice della soluzione *)
    acronimi := 42;
end;


const
    MAXA = 100;
    MAXS = 100000;
var
    A : string;
    S : string;
    fr, fw : text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, A);
    readln(fr, S);

    writeln(fw, acronimi(A, S));
    close(fr);
    close(fw);
end.

File: ed2/gara3/acronimi/att/acronimi.c
#include <stdio.h>
#include <string.h>
#include <assert.h>

int acronimi(char A[], char S[]) {
    // Per calcolare la lunghezza si puo' fare cosi'
    int lunghezza = strlen(A);

    // Mettete qui il codice della soluzione
    return 42;
}


#define MAXA 100
#define MAXS 100000

char A[MAXA + 1], S[MAXS + 1];

int main() {
    FILE *fr, *fw;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(1 == fscanf(fr, "%s", A));
    assert(1 == fscanf(fr, "%s", S));

    fprintf(fw, "%d\n", acronimi(A, S));
    fclose(fr);
    fclose(fw);
    return 0;
}
